nemo_curator/utils/distributed_utils.py

```diff
@@ -1014,6 +1018,17 @@ def get_current_client():
         return None
 
 
+def check_dask_cwd(file_list):
+    if any(not os.path.isabs(file_path) for file_path in file_list):
+        dask_cwd = list(get_current_client().run(os.getcwd).values())[0]
```

Comment:
should there be another chekc here than ensures all the workers have the same `cwd`?

Reply:
Thanks! LMK what you think.